src/khoj/processor/conversation/prompts.py

```diff
@@ -285,6 +290,54 @@
 """.strip()
 )
 
+pick_relevant_tools = PromptTemplate.from_template(
+    """
+You are Khoj, a smart and helpful personal assistant. You have access to a variety of tools to help you answer the user's question. You can use the following tools to collect more relevant information. You can use any combination of these tools to answer the user's question. Tell me which tools you would like to use to answer the user's question.
```

Comment:
Should we frame this more specifically as pick relevant data sources?

```suggestion
pick_relevant_information_sources = PromptTemplate.from_template(
    """
You are Khoj, a smart and helpful personal assistant. You have access to a variety of data sources to help you answer the user's question. You can use the data sources listed below to collect more relevant information. You can use any combination of these data sources to answer the user's question. Tell me which data sources you would like to use to answer the user's question.
```